### ISSUE TITLE
Suggestion: Support for searching in real-time 

### ISSUE THREAD

**Issue by serkanozkul** [red-gate/foundry#43]
The SQL Search in Visual Studio supports real-time searching, i.e. as we type in via the dedicated search field, the results start showing up right away.

Yet, AFAIK the SQL Search on Azure Data Studio requires the following steps;

1. Open the Command Palette
2. Select SQL Search: Search in Active Connection
3. Type in the search term
4. Repeat these steps for every search

It kills the productivity and makes the searching pain if you need to search many times. A Visual Studio like experience (if possible, a dedicated search field and easier repetitive searches) would be ideal.

**Comment by DanielPerssonProos** [comment]
I agree. This would really improve the usability for this extension for me. Alternatively I would be happy with a VS Code style icon for the extension in the left sidebar of ADS.
